packages/statemanager/src/statefulVerkleStateManager.ts

```diff
@@ -318,7 +318,7 @@ export class StatefulVerkleStateManager implements StateManagerInterface {
       chunkStems[0],
       chunkSuffixes.slice(
         0,
-        codeChunks.length <= VERKLE_CODE_OFFSET ? codeChunks.length : VERKLE_CODE_OFFSET,
+        chunkSuffixes.length <= VERKLE_CODE_OFFSET ? chunkSuffixes.length : VERKLE_CODE_OFFSET,
```

Comment:
is there a difference in code chunks and suffixes length?

Reply:
No, but it seemed more correct to actually use chunkSuffixes for this slice rather than implicitly relying that codeChunks.length === codeSuffixes.length